### PR TITLE
Us004 delete

### PR DESCRIPTION
**Pull request by tarasivanchyshyn**

### CODE REVIEW

File: src/components/Todos.tsx [hunk 1]
```diff
@@ -8,13 +8,19 @@ import classes from './Todos.module.css';
...
-  return (
-    <ul className={classes.todos}>
-      {items.map((item) => (
-        <TodoItem key={item.id} item={item}></TodoItem>
-      ))}
-    </ul>
-  );
+  let content = <p className={classes.message}>No todos yet</p>;
+
+  if (items.length > 0) {
```
Comment (Yuliia143) [inline src/components/Todos.tsx]: You can write just - if (items.length), because 0 is false

File: src/components/TodoItem.module.css [hunk 2]
```diff
@@ -9,19 +9,19 @@
...
+.input:checked {
   accent-color: #3a9190;
 }
-.item input:hover {
+.input:hover {
   cursor: pointer;
 }
 
-.text {
+.texT {
```
Comment (Yuliia143) [inline src/components/TodoItem.module.css]: Why is capital letter in the end?

File: src/components/TodoItem.tsx [hunk 3]
```diff
@@ -16,25 +16,36 @@ const TodoItem = (props: TodoItemProps) => {
...
+      <input
+        type="checkbox"
+        checked={done}
+        onChange={toggleHandler}
+        className={input}
+      />
+      <div className={`${texT} ${crossed}`}>{text}</div>
+      <div className={other}>
+        <div className={icons}>
+          <button className={crossmark} onClick={deleteHandler}>
```
Comment (Yuliia143) [inline src/components/TodoItem.tsx]: Don't you want to use svg for icon ?

File: src/components/TodoItem.module.css (hunk 2 above)
Comment (tarasivanchyshyn) [reply src/components/TodoItem.module.css]: because i have text prop in tsx

Comment (tarasivanchyshyn) [reply src/components/TodoItem.module.css]: but i will just change class name

File: src/components/TodoItem.tsx (hunk 3 above)
Comment (tarasivanchyshyn) [reply src/components/TodoItem.tsx]: I will use it for pencil. Is it a bad practice to write just a text if you can?

Comment (Yuliia143) [reply src/components/TodoItem.tsx]: Is it a letter? Because seems like some symbol

Comment (tarasivanchyshyn) [reply src/components/TodoItem.tsx]: symbol)

**Pull request merged**